src/OpenTracing/ITracer.cs

```diff
@@ -8,10 +10,17 @@ namespace OpenTracing
     public interface ITracer
     {
         /// <summary>
-        /// Returns a new <see cref="ISpanBuilder" /> for a span with the given <paramref name="operationName" />.
+        /// Starts a new Span
         /// </summary>
         /// <param name="operationName">The operation name of the span.</param>
-        ISpanBuilder BuildSpan(string operationName);
+        ISpan StartSpan(
+            string operationName,
+            IEnumerable<Tuple<string, ISpanContext>> references = null,
+            DateTimeOffset? explicitStartTime = null,
+            IEnumerable<KeyValuePair<string, string>> stringTags = null,
```

Comment:
These could be {string, object} pairs, which would remove the requirement to have many arguments for the same thing, but that would require boxing of the value types (is this a blocker though?).

Even if it is, having 4 parameters to mean the same thing is bad, and we should perhaps (if accepting this change) change to use some kind of Tag struct that can have string OR bool OR double OR int (would require either a large struct [bad] or a class with an interface - which would require boxing same as object, but be more type safe).